### ISSUE TITLE
DSDH的问题

### ISSUE THREAD

**Issue by Wangld5** [swuxyj/DeepHash-pytorch#5]
DSDH那个模型我个人觉得有问题，应该是一个epoch后更新全部的B，而不是一个batch更新一次

**Comment by swuxyj** [comment]
我按照这种做法试了下，精度还是没有提升，可能这个框架的问题，DSDH在cifar10上达不到论文说的0.82。

**Comment by Wangld5** [comment]
> 我按照这种做法试了下，精度还是没有提升，可能这个框架的问题，DSDH在cifar10上达不到论文说的0.82。

也可能是调参的问题，大佬能把这个一个epoch全部更新的方法传上来吗？

**Comment by swuxyj** [comment]
> > 我按照这种做法试了下，精度还是没有提升，可能这个框架的问题，DSDH在cifar10上达不到论文说的0.82。
> 
> 也可能是调参的问题，大佬能把这个一个epoch全部更新的方法传上来吗？

目前是在损失的forward里面调用的
`self.updateBandW(config["device"])`
 
改的话把forward里面的这行注释掉，然后再在120多行的
` for image, label, ind in train_loader:`
循环完之后加上
`criterion.updateBandW(config["device"])`
就行了，这样就算是在一个epoch全部更新了嘛，你试试呢
